igneous/tasks.py

```diff
@@ -285,20 +285,28 @@ def _compute_meshes(self):
     with Storage(self.layer_path) as storage:
       data = self._data[:, :, :, 0].T
       self._mesher.mesh(data)
```

Comment:
- It'd be more efficient to simply mask the volume so that only valid IDs are included. This will speed up the meshing and simplification steps.

Reply:
Actually nevermind, meshing is quite fast, either approach will sparsely simplify meshes.